lib/stat_tracker.rb

```diff
@@ -169,4 +180,22 @@ def lowest_scoring_home_team
     exclude = "away"
     average_goals_per_game_per_team(exclude).min_by { |team, avg| avg }[0].name
   end
+
+# ==================       Team Stats Methods      ==================
+
+  def team_info(team_id)
+    team = find_team(team_id)
+    {
+      id: team.id,
+      franchise_id: team.franchise_id,
+      name: team.name,
+      abbreviation: team.abbreviation,
+      link: team.link
+    }
+  end
+
+  def most_goals_scored(team_id)
+     goals_p_game_p_team_hash = goals_per_game_per_team
+     goals_p_game_p_team_hash[team_id].max
+  end
```

Comment:
Could this one just be `goals_per_game_per_team[team_id].max`? Instead of having to set it equal to a local variable?

Reply:
You're right! I just updated it.